src/EditorFeatures/Core.Wpf/Interactive/InteractiveDocumentNavigationService.cs

```diff
@@ -28,7 +28,7 @@ public bool CanNavigateToLineAndOffset(Workspace workspace, DocumentId documentI
         public bool CanNavigateToPosition(Workspace workspace, DocumentId documentId, int position, int virtualSpace, CancellationToken cancellationToken)
             => false;
 
-        public bool TryNavigateToSpan(Workspace workspace, DocumentId documentId, TextSpan textSpan, OptionSet options, CancellationToken cancellationToken)
+        public bool TryNavigateToSpan(Workspace workspace, DocumentId documentId, TextSpan textSpan, OptionSet options, bool allowInvalidSpan, CancellationToken cancellationToken)
```

Comment:
a general concept here is that navto can return old data from a previous session which may not be invalid.  currently TryNavigateTo can throw if someone passes in a span that isn't in teh bounds of the current doc (despite being called 'try...').  This now allows that behavior to be controlled.  By default, we still throw (so that we can catch actual logic bugs in our code).  However, in the case that we know we may truly not expect to be within the bounds of the doc anymore, we dont' want to throw as that could reasonable happen.